ads/nativo.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS-IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript} from '../3p/3p';
+
+export function nativo(global, data) {
+
+  let ntvAd;
+  (function(ntvAd, global, data) {
+    global
+      .frames
```

Comment:
Maybe I am wrong, but from what I remember when I console.log it out, global by itself doesn't get what we need. Global should be window, which should be the Top, which should have the correct location, which should get me the url of the publisher site. Is that correct.